### DISCUSSION TITLE
Why can't I add my own (or anyone else's) repos to a list?

### DISCUSSION THREAD

**Discussion by ekolis** [community/community#30240] [Other Feature Feedback, Questions, & Ideas] [answered]
This new lists feature looks nice, but I can't for the life of me figure out how to add repos to it. The instructions say I need to star them, but once I do that, how do I put them on the list?

**Comment by radar-blip** [comment, marked as answer]
Go to the starred repo you want to add, open the drop down menu, and add repo to whatever category you want to add it to.

![lists](https://user-images.githubusercontent.com/39196767/185769463-5cce86cb-a3dd-4f85-9e3f-0cc7d606208e.png)

**Reply by ekolis** [reply]
Oh, thanks! It's odd that I can't add my own repos that I haven't starred, but oh well...

**Reply by radar-blip** [reply]
You're not the only one that finds it odd. 
Maybe sometime in the near future you will be able to list unstarred repos.
Anyway, glad I could help. 
Have a great day!

**Comment by alimttth** [comment]
You're not the only one that finds it odd.
Maybe sometime in the near future you will be able to list unstarred repos.
Anyway, glad I could help.
Have a great day!
